### DISCUSSION TITLE
Unable to Connect Flutter App to Deployed Serverpod Server

### DISCUSSION THREAD

**Discussion by muradhossin** [serverpod/serverpod#2863] [Q&A] [answered]
I have successfully deployed my Serverpod server (version 2.1.2) and it's running, but I'm encountering issues connecting to it from my Flutter app (version 3.24.1). I'm getting a ServerpodClientException with a status code of 404.

Additional Information:

Serverpod Version: 2.1.2
Flutter Version: 3.24.1
Operating System: windows
Error Logs: attached
![error](https://github.com/user-attachments/assets/c141b1be-80fe-413a-8387-ff746c39cb3d)

**Comment by hampuslavin** [comment, marked as answer]
Hi @muradhossin! 
I would assume this is related to this issue, which was introduced in version `2.1.1`: https://github.com/serverpod/serverpod/issues/2856
We already merged a fix for it and will release a patch shortly 👍

**Reply by hampuslavin** [reply]
@muradhossin version `2.1.5` was just released, which hopefully contains the fix to your problem!

**Reply by muradhossin** [reply]
I have updated to version 2.1.5 and deployed the server. While the local server is working fine, I'm still encountering the same issue when trying to connect to the hosted server. Could you please guide me on how to investigate this problem and find a solution? Thank you so much for your help!

**Reply by hampuslavin** [reply]
Good that the local server works now! Just to double check, the local server didn't work after you updated, right?

To continue debugging it would be helpful to see the logs from the deployed instance. Could you paste the relevant part here, please?

**Reply by muradhossin** [reply]
Local server also works before the update. 

error:
I/flutter ( 5886): Failed call: serverpod_auth.status.getUserInfo
I/flutter ( 5886): ServerpodClientException: Not found, statusCode = 404

and can not authenticate user using email login, same setup local is working,

**Reply by hampuslavin** [reply]
Ah I see! 🤔  Do any calls at all reach the server? Is it just the `auth` calls that result in a `404`?

**Reply by muradhossin** [reply]
Yes, It is from auth. Without authentication, none of the other API calls can reach the server, so I haven’t attempted anything beyond that.

**Reply by hampuslavin** [reply]
I see. The logs you pasted above are from the client (your Flutter app). If you could paste the relevant server logs that would be helpful to continue debugging your issue.
